sources/api/migration/migrations/v1.20.0/container-runtime-metadata-nvidia/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use migration_helpers::common_migrations::{AddMetadataMigration, NoOpMigration, SettingMetadata};
+use migration_helpers::migrate;
+use migration_helpers::Result;
+use std::process;
+
+/// We added a new setting for configuring container runtime (containerd) settings only for NVIDIA k8s variants.
+fn run() -> Result<()> {
+    if cfg!(variant_family = "aws-k8s") && cfg!(variant_flavor = "nvidia") {
+        migrate(AddMetadataMigration(&[SettingMetadata {
+            metadata: &["affected-services"],
+            setting: "settings.container-runtime",
+        }]))
+    } else {
+        migrate(NoOpMigration)
+    }
+}
```

Comment:
As a rule of thumb, every change to the TOML files under "models" requires a migration, and vice versa.

I'd expect you need to add "affected-services" metadata to the corresponding toml, or the settings won't work as expected on 1.20.

Reply:
Ah, I see - the "affected-services" metadata is already present in older versions, it just doesn't do anything because the settings are unknown.